### PR TITLE
fix(mocks): Bind event node data before accessing

### PR DESCRIPTION
**Pull request by mattrobenolt**

### CODE REVIEW

**Comment by mattrobenolt** [comment]
Hi @tkaemming.

File: bin/load-mocks [hunk 1]
```diff
@@ -164,6 +164,8 @@ def create_system_time_series():
 def create_sample_time_series(event, release=None):
     if event is None:
         return
+    
+    Event.objects.bind_nodes([event], 'data')
```
Comment (macqueen) [inline bin/load-mocks]: i don't think event is imported

Comment (macqueen) [reply bin/load-mocks]: * `Event`

Comment (mattrobenolt) [reply bin/load-mocks]: Damn.

Comment (mattrobenolt) [reply bin/load-mocks]: Thanks, fixed in ef477f79f1fdbdbbc0c56990f0fceefb97b20435.

**Review by macqueen — APPROVED** [review]
🦄

**Comment by tkaemming** [comment]
> Hi @tkaemming.

Hello. 👋

**Pull request merged**